src/store/index.ts

```diff
@@ -0,0 +1,12 @@
+import { createStore, applyMiddleware } from 'redux'
+import thunkMiddleware from 'redux-thunk'
+import { composeWithDevTools } from 'redux-devtools-extension'
+import rootReducer from './reducers/rootReducer'
+
+const composedEnhancer = composeWithDevTools(
+    applyMiddleware(thunkMiddleware)
```

Comment:
Speaking of enhancers, you may use [Redux DevTools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en) extension for development build.